### PR TITLE
Updated Japanese translation file.

### PR DESCRIPTION
**Pull request by toddcarnes**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,7 @@ ui_*.h
 *.pro.user*
 qrc_*
 docs/html
+.directory
```
Comment (bjorn) [inline .gitignore]: Why?

**Comment by bjorn** [comment]
@teraapi Could you review this update, please?

File: .gitignore (hunk 1 above)
Comment (toddcarnes) [inline .gitignore]: Because it was pulling in the .directory for on my system.

Comment (bjorn) [inline .gitignore]: What is the .directory?

Comment (toddcarnes) [inline .gitignore]: A normally invisible file that the file manager on kubuntu used to track
files in the directory.

Comment (bjorn) [inline .gitignore]: Ok, then please set this as a global ignore, for example by putting it in ~/.gitignore and doing:

    git config --global core.excludesfile ~/.gitignore

Comment (toddcarnes) [inline .gitignore]: I will do that. Thank you for the information. I wasn't aware I could do
that. :)

Comment (bjorn) [inline .gitignore]: You're welcome. :-)

**Comment by teraapi** [comment]
Looks good to me. :))

**Comment by bjorn** [comment]
Alright, thanks @teraapi for the review and @toddcarnes for the update. I've applied the translation in commit 2de09c5aa6d2067efa9bf37fdffda5dd128ddae6 (dropping the change to .gitignore).

**Pull request closed without merging**